rllib/agents/dqn/dqn.py

```diff
@@ -204,32 +223,27 @@ def execution_plan(
 
         def update_prio(item):
             samples, info_dict = item
-            if config.get("prioritized_replay"):
-                prio_dict = {}
-                for policy_id, info in info_dict.items():
-                    # TODO(sven): This is currently structured differently for
-                    #  torch/tf. Clean up these results/info dicts across
-                    #  policies (note: fixing this in torch_policy.py will
-                    #  break e.g. DDPPO!).
-                    td_error = info.get(
-                        "td_error", info[LEARNER_STATS_KEY].get("td_error")
-                    )
-                    samples.policy_batches[policy_id].set_get_interceptor(None)
-                    batch_indices = samples.policy_batches[policy_id].get(
-                        "batch_indexes"
+            prio_dict = {}
```

Comment:
What happens if the user specifies a non-prio-replay buffer in their config? Would this be handled correctly by the new code below?